pkgs/top-level/python-packages.nix

```diff
@@ -5043,6 +5043,8 @@ self: super: with self; {
 
   korean-lunar-calendar = callPackage ../development/python-modules/korean-lunar-calendar { };
 
+  krb5 = callPackage ../development/python-modules/krb5 { };
```

Comment:
Any chance we could call this `pykrb5` to align with the repo name and avoid clashes with `pkgs.krb5`?